@types/config.ts

```diff
@@ -3,9 +3,8 @@ export interface Geocoder {
   useLang: boolean;
   maxItems: number;
   useFocus: boolean;
-  focusPrecision: string;
+  focusPrecision: Array<[string, string]>;
```

Comment:
I think instead of `[string, string]`, the array sould contain an object with a *zoom* and a *precision* field (according to the default in *default_config.yml*).